torch/_inductor/cudagraph_trees.py

```diff
@@ -969,7 +969,7 @@ def reconstruct_outputs(self):
         if not self.cached_tensor_outputs:
             self._initialize_cached_tensors()
 
-        outputs: List[torch.Tensor] = []
+        outputs: List[Any] = []
```

Comment:
nit: `List[Union[int, None, torch.Tensor]]`

Reply:
IIRC this created a lot of nullcheck issues but let me try again